src/Components/MyBooks/MyBooks.css

```diff
@@ -18,6 +18,15 @@
 .user-book-welcome {
   color: #0F4D92;
   font-size: xx-large;
-  font-family: Verdana, Geneva, Tamhoma, sans-serif;
+}
+
+@media screen and (max-width: 920px) {
+  .my-books-container {
+    display: flex;
+    flex-direction: column;
+    width: fit-content;
+    align-items: center;
+  }
+
```

Comment:
@726px, it should stack over each other like it does on the home page so it avoids the break

Reply:
![Screenshot 2023-02-15 at 5 24 53 PM](https://user-images.githubusercontent.com/108101979/219226373-797feeac-bb17-40ca-9daf-aa51f4ba17fd.png)

Reply:
potentially a flex-wrap: wrap

Reply:
Fixed!

Reply:
Wait this is so weird... that's not the design that I have on my branch